deploy/crds/db.orange.com_cassandraclusters_crd.yaml

```diff
@@ -207,6 +207,13 @@ spec:
                   - memory
                   type: object
               type: object
+            restartCountBeforePodDeletion:
+              description: RestartCountBeforePodDeletion defines number of restart
```

Comment:
defines the number of restarts allowed for a cassandra container before deleting the pod to force its restart from scratch. If set to 0...